doc/source/reference/c-api.array.rst

```diff
@@ -3464,11 +3464,11 @@ Other constants
 
 .. c:var:: NPY_MAXDIMS
 
-    The maximum number of dimensions allowed in arrays.
+    The maximum number of dimensions allowed in statically allocated arrays.
```

Comment:
It is true, that `NPY_MAXDIMS` is mostly used for statically allocated data. But the first version was correct, it is also the maximum number of dimensions to an array (which is why it can be used for the statically allocated data).

Reply:
Yeah, I don't think this is an improvement. `NPY_MAXDIMS` is used _internally_ for statically-allocated _dimension_ **C**-arrays, but it applies to regular `ndarray` objects.